src/hermes_claude_agent_sdk/parity/v4_live_packets.py

```
@@ -0,0 +1,399 @@
+        if digest in seen:
+            raise V4LivePacketViolation("event projections are reused")
+        seen.add(digest)
+        if type(item["byte_length"]) is not int or not 1 <= item["byte_length"] <= 1_048_576:
+            raise V4LivePacketViolation("event projection byte length is invalid")
+        counts[raw_kind] = counts.get(raw_kind, 0) + 1
+        if raw_kind.casefold() in _CONTENT:
+            if item["terminal_status"] is not None:
+                raise V4LivePacketViolation("content projection carries a terminal")
+            content.append(digest)
+            continue
+        kind = _kind(raw_kind); status = item["terminal_status"]
```

Ruff (0.16.3)

[error] Multiple statements on one line (semicolon)

(E702)